docs/DeveloperGuide.adoc

```diff
@@ -334,22 +823,140 @@ Use case ends.
+* 2a. System does not find any athletes satisfying all the constraints.
++
+[none]
+** 2a1. System shows an empty list.
+
+[discrete]
+==== Use case: UC13 - View a specific participant
+
+*MSS*
+
+1. User obtains a list of athletes by listing (<<Use case: UC11 - View my participant list, *UC11*>>).
+or searching (<<Use case: UC12 - Search for participants, *UC12*>>).
+2. User requests to view more information on a specific athlete in the list.
+3. System shows detailed information of the book.
++
+Use case ends.
+
+*Extensions*
+
+[none]
+* 2a. System does not find the athlete in the list.
++
+Use case restarts from step 1.
+
+* 2b. User gives an invalid index.
++
+[none]
+** 2b1. System shows an error message.
++
+Use case resumes at step 2.
+
+Use case ends.
+
+[discrete]
+==== Use case: UC14 - Edit a specific participant
+
+*MSS*
+
+1. User views a specific athlete (<<Use case: UC13 - View a specific participant, *UC13*>>)
+2. User requests certain data about the athlete to be updated.
+3. System shows success message.
++
+Use case ends.
+
+*Extensions*
+
+* 2a. User gives invalid data.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 2a1. System shows an error message.
 +
 Use case resumes at step 2.
 
+[discrete]
+==== Use case: UC15 - Disqualify a specific participant
+
+*MSS*
+
+1. User views a specific participant (<<Use case: UC13 - View a specific participant, *UC13*>>)
+2. User requests for a particular athlete to be disqualified from the competition.
+3. System shows success message.
++
+Use case ends.
+
+[discrete]
+==== Use case: UC16 - Call for next participant in lifting order
+
+*MSS*
+
+1. User requests for next athlete to attempt lift.
+2. System shows athlete that is lifting next.
+3. User updates system after athlete attempts lift.
+4. System shows success message.
++
+Use case ends.
+
+*Extensions*
+
+* 2a. No more participants in lifting order.
++
+Use case ends.
+
+[discrete]
+==== Use case: UC16 - Exit competition
+
+*MSS*
+
+1. User requests to exit competition mode.
+2. System exits competition mode.
++
+Use case ends.
+
+==== Use case: UC16 - Finalize competition
+
+*MSS*
+
+1. User requests to finalize competition data.
+2. System finalizes competition and prevents any further updating of the competition data.
++
+Use case ends.
+
+*Extensions*
+
+* 2a. There are still participants that have not finished their lifts
++
+
+Use case ends.
+
 _{More to be added}_
 
+
+
 [appendix]
 == Non Functional Requirements
 
-.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+.  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `1.8.0_60` or higher installed.
+.  Should work on both 32-bit and 64-bit environments.
+.  Should come with automated unit tests and open source code.
+.  Should come with a user guide and a developer guide.
```

Comment:
These NFRs do not have anything to do with the system and they do not specify the constraints under which system is develeoped and operated.